terasoluna-gfw-parent/pom.xml

```diff
@@ -333,6 +333,26 @@
             </dependency>
             <!-- == End Spring Boot == -->
 
+            <!-- == Begin Spring Framework == -->
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-framework-bom</artifactId>
+                <version>${org.springframework.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <!-- == End Spring Framework == -->
```

Comment:
- [ ] You have to write it before the BOOT definition to reflect it.
When modifying dependencies, make sure they are reflected in `mvn dependency:tree.`

Reply:
4c972cd
Moved the type:pom and scope:import dependencies to before the BOOT definition as noted.
Also added a dependency about tomcat.